README.md

```diff
@@ -26,7 +35,7 @@ This is a known bug in PyTube <br>
 ## Dependencies
 - **PyQt5==5.14.1**
 - **PyQt5-sip==12.7.0** `extension module provides support for the PyQt5 package`
-- [**pytube==9.5.3**](https://github.com/nficano/pytube) `A lightweight, Pythonic, dependency-free, library for downloading YouTube Videos`
+- [**pytube3**](https://github.com/hbmartin/pytube3) `A lightweight, Pythonic, dependency-free, library for downloading YouTube Videos` - hbmartin forked version
```

Comment:
I'd add here it's a fork from original pytube (yeah I know, but, better to still mention that, as the creator might notice how far this issue has been widespread and WHY their usage stats are going down, amirite?)